sdk/containerregistry/Azure.Containers.ContainerRegistry/tests/ContainerRegistryBlobClientLiveTests.cs

```diff
@@ -0,0 +1,279 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Azure.Containers.ContainerRegistry.Specialized;
+using Azure.Core.TestFramework;
+using NUnit.Framework;
+
+namespace Azure.Containers.ContainerRegistry.Tests
+{
+    [NonParallelizable]
+    public class ContainerRegistryBlobClientLiveTests : ContainerRegistryRecordedTestBase
```

Comment:
Do we have a sample in a test or elsewhere showing the steps it takes to upload a blob end-to-end and download a blob end-to-end?  I think this would be good to be able to look at in order to [evaluate the APIs](https://github.com/dotnet/runtime/blob/main/docs/coding-guidelines/framework-design-guidelines-digest.md#scenario-driven-design).

Reply:
Yes, that's the `CanUploadBlob()` test in this file. I will add samples before the archboard review.